driver.py

```diff
@@ -65,6 +65,10 @@ def parse_options():
                         metavar='SEC', default=180, type=int,
                         help='specify the timeout (default: %(default)s sec)')
 
+    parser.add_argument('--coverage',
+                        metavar='SERVER_ADDRESS(HOST:PORT)',
+                        help='use jerry-debugger to calculate the JS source code coverage')
```

Comment:
Please also define the `--coverage` option in the README.md file ([Testrunner parameters](https://github.com/Samsung/js-remote-test#testrunner-parameters) section).